Source/m_menu.c

```diff
@@ -3710,6 +3713,37 @@ void static M_SmoothLight(void)
   P_SegLengths(true);
 }
 
+static void M_UpdateStrictModeItems(void)
+{
+  if (strictmode)
+  {
```

Comment:
How about `S_STRICT` that acts like `S_DISABLE` but only in strict mode?

Reply:
I don't know why this should help?

Reply:
`if (flags & S_DISABLE || STRICTMODE(flags & S_STRICT))`

Reply:
I like `M_UpdateStrictModeItems` better because we have all the menu options in one place.

Reply:
Okay, fine with me.